### PR TITLE
Add LTI display name

### PR DESCRIPTION
**Pull request by RiteshChouhan21**
@dcadams please review, this PR will show lti display name for all the blocks.

### CODE REVIEW

**Comment by RiteshChouhan21** [comment]
@dcadams item's order is updated.

**Comment by dcadams** [comment]
I made these changes on my Tutor devstak and they look good.

:+1:

**Review by dcadams — CHANGES_REQUESTED** [review]
Looks good. Just a couple of nits.

File: lti_params_api/patch/course_experience_utils_custom.py [hunk 1]
```diff
@@ -212,6 +212,7 @@ def recurse_mark_auth_denial(block):
         requested_fields=[
             'children',
             'display_name',
+            'lti_display_name',
             'type',
```
[the remark this answers is not in the document]
Comment (arpitgandhi-xoriant) [reply lti_params_api/patch/course_experience_utils_custom.py]: Done

File: tests/tests.py [hunk 2]
```diff
@@ -60,10 +61,10 @@ def test_lti_params(self):
         query_string = '/api/lti_params_api/lti_params_list/?course_id={0}'.format(self.course_id)
         response = client.get(query_string)
         data = response.data[0]
-
-        self.assertIn('i4x://edX/100/lti_advantage_consumer/lti_advantage_consumer_', data['block_key'])
+        self.assertIn('i4x://edX/100/lti_advantage_consumer/knowledge_check', data['block_key'])
```
[the remark this answers is not in the document]
Comment (arpitgandhi-xoriant) [reply tests/tests.py]: Done

**Review by dcadams — APPROVED** [review]
:+1: 
Please squash and merge.

**Comment by arpitgandhi-xoriant** [comment]
Squashed and merged into master.

**Pull request closed without merging**